### ISSUE TITLE
dartdoc shows warnings when defining the props getter in a EquatableMixin

### ISSUE THREAD

**Issue by llucax** [felangel/equatable#80]
**Describe the bug**
Using `dartdoc` to generate docs for a class implemented `with EquatableMixin` shows a warning about `[Equatable]` not being defined in the inherited documentation for `props`.

**To Reproduce**

1. `flutter create eq` (probably not necessary to reproduce but it was a quick way to get a new project to use equatable).
2. `cd eq`
3. Edit `pubspec.yaml` and add `equatable:` to the `dependencies:` section.
4. Run `flutter pub get`
5. Edit `lib/main.dart` and replace the content with:
   ```dart
   import 'package:equatable/equatable.dart';
  
   /// Test class.
   class E with EquatableMixin {
     /// Test bool.
     bool b;
     @override
     List<Object> get props => [b];
   }
  
   void main() {
     final e = E();
     e.b = true;
   }
   ```
7. Run `dartdoc`

**Expected behavior**
The documentation should be generated without any warnings.

**Screenshots**
Not exactly a screenshot, but the `dartdoc` output:
```console
$ dartdoc 
Documenting eq...
Initialized dartdoc with 56 libraries in 19.4 seconds
Generating docs for library main from package:eq/main.dart...
  warning: unresolved doc reference [Equatable]
    from main.E.props: (file:///home/luca/devel/flutter/eq/lib/main.dart:6:20)
    in documentation inherited from equatable_mixin.EquatableMixin.props: (file:///home/luca/.pub-cache/hosted/pub.dartlang.org/equatable-1.2.3/lib/src/equatable_mixin.dart:12:20)
Validating docs...
found 1 warning and 0 errors
Documented 1 public library in 4.0 seconds
Success! Docs generated into /home/luca/devel/flutter/eq/doc/api
```

**Version**
Dart SDK version: 2.9.1 (stable) (Wed Aug 12 12:37:19 2020 +0200) on "linux_x64"
equatable 1.2.3
dartdoc version: 0.32.1

**Additional context**
Not really.

**Comment by felangel** [comment]
Hi @llucax 👋 
Thanks for opening an issue!

I believe this should have been fixed in v1.2.4. Can you please try upgrading and let me know if you're still seeing the warning?  Thanks 👍

**Comment by llucax** [comment]
Hi @felangel, sadly it is still happening in 1.2.4:
```console
$ dartdoc
Documenting eq...
Initialized dartdoc with 58 libraries in 16.9 seconds
Generating docs for library main from package:eq/main.dart...
  warning: unresolved doc reference [Equatable]
    from main.E.props: (file:///home/luca/devel/flutter/eq/lib/main.dart:8:20)
    in documentation inherited from equatable_mixin.EquatableMixin.props: (file:///home/luca/.pub-cache/hosted/pub.dartlang.org/equatable-1.2.4/lib/src/equatable_mixin.dart:12:20)
Validating docs...
found 1 warning and 0 errors
Documented 1 public library in 0.7 seconds
Success! Docs generated into /home/luca/devel/flutter/eq/doc/api
$ flutter pub deps | grep equatable
|-- equatable 1.2.4
```

**Comment by felangel** [comment]
@llucax thanks! This should be resolved in v1.2.5 🎉

**Comment by llucax** [comment]
Thanks! That did the trick!
